src/components-barnes-toolkit/scss/setup/_variables.scss

```diff
@@ -108,5 +126,13 @@ $headerHeightsShopify: (
   large: 110px,
   medium: 110px,
   small: 70px,
-  xsmall: 70px
+  xsmall: 70px,
+);
+
+// Centennial Logo Padding
+$logo100Padding: (
+  xxlarge: 60px,
```

Comment:
should this also be changed to 30px? Otherwise the logo will "jump" when you transition past that breakpoint!